scripts/data_recorder_playback_demo.py

```diff
@@ -1,28 +1,69 @@
-import pickle
 from pathlib import Path
 
+import rerun as rr
+
+from conq.logging.replay.conq_log_file import ConqLog
 from conq.cameras_utils import image_to_opencv
 
+RR_TIMELINE_NAME = "stable_time"
+NANOSECONDS_TO_SECONDS = 1e-9
+
+
+def get_log_paths():
+    """Do all the path manipulation necessary to point to log and metadata files
+
+    This is just a convenience function to make the main function less cluttered.
+    """
+    root_path = Path("/media/big_narstie/datasets/conq_hose_manipulation_raw/")
+
+    exp_name = "conq_hose_manipulation_data_1700079751"
+    # exp_name = "conq_hose_manipulation_data_1700080718"
+    # exp_name = "conq_hose_manipulation_data_1700081021"
+    # exp_name = "conq_hose_manipulation_data_1700082856"
+    exp_path = root_path / exp_name
```

Comment:
[OPTIONAL] Will we always use the same `exp_name`? If not, please consider passing this variable to the function or from a config.